src/loaders/helpers.js

```diff
@@ -23,8 +23,9 @@ module.exports.hasLoader = function hasLoader(rule, loaderName) {
       const loader = rule.use[i];
       // check exact name, eg "url-loader" or its path "node_modules/url-loader/dist/cjs.js"
       if (
-        loader.loader === loaderName ||
-        loader.loader.includes(`/${loaderName}/`)
+        (typeof loader.loader === 'string' &&
+        loader.loader === loaderName )||
+       (Array.isArray(loader.loader) && loader.loader.includes(`/${loaderName}/`))
```

Comment:
`use[*].loader` cannot be an array. It's a string. So proposed changes are incorrect.

A comment above if clause explains what happening `check exact name, eg "url-loader" or its path "node_modules/url-loader/dist/cjs.js"` - `includes` checks presence of substring eg. for `url-loader` it will search `/url-loader/`.

If you have real bugous case - please provide additional information.